frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -280,21 +282,33 @@ export class DragElementService {
                     // This ratio has been determined by trial and error
                     const height = SimulatedRegion.image.height / 23.5;
                     const width = height * SimulatedRegion.image.aspectRatio;
-                    const simulatedRegion = SimulatedRegion.create(
-                        {
-                            x: position.x - width / 2,
-                            y: position.y + height / 2,
-                        },
-                        {
-                            height,
-                            width,
-                        },
-                        'Einsatzabschnitt ???'
+                    const simulatedRegion = cloneDeepMutable(
```

Comment:
It has to: Because I have to create a simulated region and a transfer point, which both have to bidirectionally reference each other. It is not going to be mutable anymore (i think), once it gets added to the state.